docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -1,32 +1,52 @@
-The solution is to provide input shapes of a static rank explicitly.
+The parameter `--input_shape` allows to override original input shapes to ones compatible for a given model.
+Dynamic shapes, i.e. with dynamic dimensions, in the original model can be replaced with static shapes for the converted model, and vice versa.
+The dynamic dimension can be marked in Model Optimizer command-line as `-1` or `?`.
+For example, launch the Model Optimizer for the ONNX* OCR model and specify dynamic batch dimension for inputs.
 
-## When to Specify --static_shape Command Line Parameter
-If the `--static_shape` command line parameter is specified the Model Optimizer evaluates shapes of all operations in the model (shape propagation) for a fixed input(s) shape(s). During the shape propagation the Model Optimizer evaluates operations *Shape* and removes them from the computation graph. With that approach, the initial model which can consume inputs of different shapes may be converted to IR working with the input of one fixed shape only. For example, consider the case when some blob is reshaped from 4D of a shape *[N, C, H, W]* to a shape *[N, C, H \* W]*. During the model conversion the Model Optimize calculates output shape as a constant 1D blob with values *[N, C, H \* W]*. So if the input shape changes to some other value *[N,C,H1,W1]* (it is possible scenario for a fully convolutional model) then the reshape layer becomes invalid.
-Resulting Intermediate Representation will not be resizable with the help of OpenVINO Runtime API.
+```sh
+mo --input_model ocr.onnx --input data,seq_len --input_shape [-1,150,200,1],[-1]
+```
 
-## Parameters for Pre-Processing
-Input data may require pre-processing such as `RGB<->BGR` conversion and mean and scale normalization. To learn about Model Optimizer parameters used for pre-processing, refer to [Optimize Preprocessing Computation](../Additional_Optimizations.md).
+## When to Specify `--static_shape` Command Line Parameter
+If the `--static_shape` command line parameter is specified, the Model Optimizer evaluates shapes of all operations in the model (shape propagation)
+for a fixed input(s) shape(s). During the shape propagation the Model Optimizer evaluates operations *Shape* and removes them from the computation graph.
+With that approach, the initial model which can consume inputs of different shapes may be converted to IR working with the input of one fixed shape only.
+For example, consider the case when some blob is reshaped from 4D of a shape *[N, C, H, W]* to a shape *[N, C, H \* W]*.
+During the model conversion the Model Optimize calculates output shape as a constant 1D blob with values *[N, C, H \* W]*.
+So if the input shape changes to some other value *[N, C, H1, W1]* (it is possible scenario for a fully convolutional model) then the reshape layer
+becomes invalid. Resulting Intermediate Representation will not be resizable with the help of OpenVINO Runtime API.
```

Comment:
Too low-level description. Example is cryptic. Please rewrite the whole thing focusing on: use case when it provides value (smaller IR and shorted load time), and limitation -- inability to change shape later in run-time. Describe why just --input_shape is not enough.

Reply:
Use when data size is determined and would be changed. User can freeze shapes of a model.
Values:
 - compact model
 - improved loading time

Reply:
Done